liquibase-standard/src/main/java/liquibase/changelog/StandardChangeLogHistoryService.java

```diff
@@ -244,6 +247,14 @@ public void init() throws DatabaseException {
                 }
             }
 
+            if (hasShortChecksum) {
+                executor.comment("Modifying size of databasechangelog.md5sum column");
+                statementsToExecute.add(new ModifyDataTypeStatement(getLiquibaseCatalogName(),
+                        getLiquibaseSchemaName(), getDatabaseChangeLogTableName(), "MD5SUM",
+                        charTypeName + "(" + MD5_COLUMN_SIZE + ")"));
+
+            }
```

Comment:
I wonder how many databases support this type of type migration. I don't know offhand, but I'm pretty sure it isn't ALL databases. I think some you have to do some data copying nonsense. Or does Liquibase handle that for you?